jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC

```diff
@@ -33,25 +31,21 @@ export EXT=""
 ##############################################
 # Define COM directories
 ##############################################
-export COMIN=${COMIN:-$(compath.py ${envir}/${NET}/${gfs_ver})/${RUN}.${PDY}/${cyc}/${COMPONENT}}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${gfs_ver}/${NET}.${PDY})/${cyc}/${COMPONENT}/gempak}
-
-export SENDDBN=${SENDDBN:-NO}
-export DBNROOT=${DBNROOT:-${UTILROOT}/fakedbn}
-
-mkdir -m 775 -p ${COMOUT}
-
+export COMIN="${COMIN:-$(compath.py "${envir}"/"${NET}"/"${gfs_ver}")/${RUN}.${PDY}/${cyc}/${COMPONENT}}"
+export COMOUT="${COMOUT:-$(compath.py -o "${NET}"/"${gfs_ver}"/"${NET}"."${PDY}")/${cyc}/${COMPONENT}/gempak}"
 
-# TODO - Assess what is going on with overwriting $DATA here (#1224)
+export SENDDBN="${SENDDBN:-NO}"
+export DBNROOT="${DBNROOT:-${UTILROOT}/fakedbn}"
 
-export DATA_HOLD=${DATA}
+mkdir -m 775 -p "${COMOUT}"
 
 #################################################################
 # Execute the script for the regular grib
 #################################################################
-export DATA=${DATA_HOLD}/SPECIAL
-mkdir -p ${DATA}
-cd ${DATA}
+export DATA_SPECIAL="${DATA}/SPECIAL"
```

Comment:
I am not sure what the `exscript` here is doing or how it is handling data (and `DATA`).
All told, I don't think this is per EE2 either way.

Reply:
FYI, it seems to be bigger than just what the title says.

Reply:
@aerorahul ^^^ I apologize, I don't follow the last comment.